### PR TITLE
feat: add plugin notification

### PR DESCRIPTION
**Pull request by asterwyx**
Add plugin notification to show notification center. Notification is a
fixed plugin.

Log: add plugin notification
Issue: https://github.com/linuxdeepin/developer-center/issues/6695

### CODE REVIEW

File: plugins/notification/notificationplugin.cpp [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+const QString NotificationPlugin::pluginName() const
+{
+    return "notification";
+}
+
+const QString NotificationPlugin::pluginDisplayName() const
+{
+    return tr("Notification");
```
Comment (tsic404) [inline plugins/notification/notificationplugin.cpp:38]: 翻译文件更新？

Comment (asterwyx) [reply plugins/notification/notificationplugin.cpp:38]: Done

**Review by tsic404 — APPROVED** [review]

**Comment by tsic404** [comment]
image
是不是没有做？

**Review by tsic404 — CHANGES_REQUESTED** [review]
image

**Review by tsic404 — APPROVED** [review]

**Pull request merged**